spock-specs/src/test/groovy/org/spockframework/smoke/condition/SetComparisonRendering.groovy

```diff
@@ -0,0 +1,111 @@
+package org.spockframework.smoke.condition
+
+class SetComparisonRendering extends ConditionRenderingSpec {
+
+  def "Set differences are displayed"() {
+    expect:
+    isRendered('''
+a == b
+| |  |
+| |  [2, 3, 4]
+| false
+| 2 differences (66% similarity missing 1 extra 1)
```

Comment:
separate with comma and stick to one order(value, description): `(66% similarity, 1 missing, 1 extra)`